hooks/image/50-xkb-layout.chroot

```diff
@@ -21,4 +22,17 @@ if [ -n "${EIB_IMAGE_XKB_LAYOUT}" ]; then
 	cat <<- EOF >> ${TARGET_PATH}/00-keyboard.conf
 	EndSection
 	EOF
+
+	# Write to /etc/default/keyboard too, Debian patches
+	# localed to read and write from this location
+
+	cat <<- EOF >> ${DEBIAN_DEFAULTS_PATH}/keyboard
+		XKBLAYOUT=${EIB_IMAGE_XKB_LAYOUT}
+	EOF
```

Comment:
Very nitpicky, but the `<<-` will strip the indent up to where the `EOF` is. That would cause `XKBLAYOUT` and `XKBVARIANT` to be indented 2 spaces, which I don't think was intended.

Reply:
Indeed, fixed!